docs/internal/instrumentation-libraries.md

```diff
@@ -0,0 +1,117 @@
+| [GraphQL](https://www.nuget.org/packages/GraphQL/) | Yes | | The current instrumentation needs updates to match the semantic conventions. |
+| [GraphQL Client](https://github.com/graphql-dotnet/graphql-client) | | | |
+| [RabbitMQ](https://www.nuget.org/packages/RabbitMQ.Client) | | | These is an [issue in the RabbitMQ repo](https://github.com/rabbitmq/rabbitmq-dotnet-client/issues/776) to add instrumentation directly into RabbitMQ. |
+| [Kafka](https://www.nuget.org/packages/Confluent.Kafka) | | | |
+| [NServiceBus](https://docs.particular.net/nservicebus/) | | | |
+| [MassTransit](https://masstransit-project.com/) | | | |
+
+### Logging
+
+TODO: Determine how we want to support logging.
+
+1. [Log4net](https://logging.apache.org/log4net/)
+1. [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging/)
+1. [Serilog](https://github.com/serilog/serilog)
+1. [NLog](https://github.com/NLog/NLog)
+1. [Common.Logging](https://github.com/net-commons/common-logging)
+
+### Other
+
+| Library | [Tracing Support](../config.md#instrumented-traces-libraries-and-frameworks) | [Metrics Support](../config.md#instrumented-metrics-libraries-and-frameworks) | Notes |
+| --- | :---: | :---: | --- |
+| [Microsoft.Extensions.Caching](https://docs.microsoft.com/en-us/dotnet/core/extensions/caching) | | | TODO: Evaluate if this is desired. |
+
+## Cloud Environments and SDKs
+
+### [AWS](https://aws.amazon.com/)
+
+TODO: Determine appropriate resource detectors and AWS SDK related
+instrumentation libraries to support.
+
+### [Azure](https://azure.microsoft.com/)
+
+TODO: Determine appropriate resource detectors and Azure SDK related
+instrumentation libraries to support. This should also include support
+for frameworks like [Azure Service Fabric](https://azure.microsoft.com/en-us/services/service-fabric/).
+
+### [GCP](https://cloud.google.com/)
+
+TODO: Determine appropriate resource detectors and GCP SDK related
+instrumentation libraries to support.
+
+### Libraries we should not support
+
+| Library | Notes |
+| --- | --- |
+| [System.IO.Pipelines](https://docs.microsoft.com/en-us/dotnet/standard/io/pipelines) | This library is used for high-throughput scenarios and instrumenting this library could potentially impact the performance of the application in a negative manner. |
+| [Pipelines.Sockets.Unofficial](https://github.com/mgravell/Pipelines.Sockets.Unofficial) | This is a lower-level networking library and instrumenting it can affect the performance of an application in negative ways causing an increase in network traffic (as compared to the application running without instrumentation). |
+| [System.Threading.Channels](https://docs.microsoft.com/en-us/dotnet/api/system.threading.channels) | This is a lower-level library and may not provide enough contextual information on its own. |
+| [Cassandra](https://www.nuget.org/packages/CassandraCSharpDriver) | Lower usage |
```

Comment:
Should it be `Low usage` instead of `Lower usage`? If so then the same applies to other places.